rai/node/node.cpp

```diff
@@ -2971,6 +2971,15 @@ void rai::add_node_options (boost::program_options::options_description & descri
 bool rai::handle_node_options (boost::program_options::variables_map & vm)
 {
 	auto result (false);
+	boost::filesystem::path data_path;
+	if (vm.count ("data_path"))
+	{
+		data_path = boost::filesystem::path (vm ["data_path"].as <std::string> ());
```

Comment:
This would be better as a ternary IMO.

Reply:
I agree with @androm3da, this would be cleaner as a ternary. This change also makes the data_path assignment under the 'vacuum' option redundant, and it should be removed.